tests/test_profile_general.py

```diff
@@ -406,14 +593,34 @@ def test_path():
     else:
         assert sorted(list(file_dict.keys())) == ALL_CSVS
 
-    if config["COUNTER_LOGGING"]:
-        log_counter(file_dict, inspect.stack()[0][3])
-
-    if config["METRIC_LOGGING"]:
-        log_metric(
-            inspect.stack()[0][3],
-            {"default": {"absolute": DEFAULT_ABS_DIFF, "relative": DEFAULT_REL_DIFF}},
-        )
+    logging(
+        inspect.stack()[0][3],
+        workload_dir,
+        file_dict,
+        {
+            "default": {"absolute": DEFAULT_ABS_DIFF, "relative": DEFAULT_REL_DIFF},
+            "2.1.22": {"absolute": 0, "relative": 5},
+            "2.1.27": {"absolute": 0, "relative": 5},
+            "5.1.5": {"absolute": 5, "relative": 0},
+            "5.2.8": {"absolute": 1, "relative": 0},
+            "5.2.11": {"absolute": 2, "relative": 0},
+            "6.1.1": {"absolute": 0, "relative": 6},
+            "6.1.2": {"absolute": 0, "relative": 5},
+            "13.1.0": {"absolute": 1, "relative": 0},
+            "14.1.0": {"absolute": 0, "relative": 5},
+            "16.1.2": {"absolute": 1, "relative": 0},
+            "16.2.0": {"absolute": 0, "relative": 7},
+            "16.3.4": {"absolute": 0, "relative": 5},
+            "17.1.2": {"absolute": 0, "relative": 3},
+            "17.2.3": {"absolute": 1, "relative": 0},
+            "17.2.14": {"absolute": 5, "relative": 0},
+            "17.3.14": {"absolute": 1, "relative": 0},
+            "17.3.15": {"absolute": 1, "relative": 0},
+            "17.3.16": {"absolute": 2, "relative": 0},
+            "18.1.13": {"absolute": 1, "relative": 0},
+            "18.1.16": {"absolute": 1, "relative": 0},
```

Comment:
I think it would be useful to try and minimize how much metric/tolerance override is needed when calling the updated `logging` function.  Ideally, we would setup the global list of metrics to test once with tolerances that are mostly usable across multiple tests and override more sparingly.  Then, most checks would hopefully be shortened to look more like:

```
  logging(
        inspect.stack()[0][3],
        workload_dir,
        file_dict,
        override=None
}
```